public/js/main-template.js

```diff
@@ -12,93 +12,124 @@ $(document).ready(function () {
     asideBarRight = document.getElementById("AsideBarRight");
     fieldContainer = document.getElementById("field-container");
     installcontainer = document.getElementById("install-container");
+   if (document.getElementsByClassName("PannelIcon")[0]) {
+        document.getElementsByClassName("PannelIcon")[0].classList.toggle("Active");
+        document.getElementsByClassName("PannelIcon")[0].parentElement.classList.toggle("RightPanel1Active");
+        document.getElementsByClassName("RightPanel1")[0].classList.toggle("Active");
 
-    // $(".LeftPannelIcon1").click(function () {
-    //     toggleInstallMenu();
-    // });
-
-    // $(".PannelIcon2").click(function () {
-    //     toggleRightInstallMenu();
-    // });
-  
-    
+    }
 });
 
-function PannelIcon2(event)  {
-    $(".AsideBarInstallRight").toggleClass("RightPanel2Active");
-    $(".RightPanel2").toggleClass("Active");
-    event.currentTarget.classList.toggle("Active")
-};
-
-function toggleRightInstallMenu() {
-    $(this).toggleClass("Active", { duration: 3000 });
-    $(".PannelIcon").removeClass("Active", { duration: 3000 });
-    $(".AsideBarInstallRight").toggleClass("RightPanel2Active");
-    $(".AsideBarRight").removeClass("RightPanel1Active");
-    $(".RightPanel1").removeClass("Active");
-    $(".RightPanel2").toggleClass("Active");
-
-    if ($(".AsideBarRight").hasClass("Active")) {
-        toggelProjectSelectDiv();
-        removeClasses([rightMenuBtn], "display-block");
-        addClasses([rightMenuBtn], "display-none");
+function toggelProjectSelectDiv(event, page) {
+
+    if (event != "") {
+        if (event.currentTarget.classList.contains("PannelIcon")) {
+            if (document.getElementsByClassName("RightPanel2")[0].classList.contains("Active")) {

```

Comment:
same here as well add a unique id to the element and access like. would be good to change all over the app this way.
document.getElementsById("Right-Panel-two").classList.contains("Active")